ConsoleApp/ConsoleApp/CatFact.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ConsoleApp
+{
+    internal class CatFact
+    {
+     //   {"status":
```

Comment:
it's bad style to leave comments like this here, better remove it and create **example.json** with response example